invenio_orcid/tasks.py

```diff
@@ -128,16 +141,20 @@ def send_to_orcid(sender, api=None):
                     else:
                         api.update_record(author_orcid, token,
                                           'work', orcid_json, str(put_code))
-                    logger.info("Succersfully sent " +
-                                sender.get('control_number') + " to orcid.")
+                    current_app.logger.info(
+                        "Succersfully sent {0} to orcid.").format(
```

Comment:
@Panos512 my comment was about using consistently single quotes and string formatting. For everything else just follow PEP8, file context and common sense with the aim of maximum code readability.